docs/journey/day1/README.md

```diff
@@ -19,7 +19,6 @@ You need to have the following installed in the OCP 4.x cluster.
 
 And, you will need this.
 
-* Create your [GitOps repository](prerequisites/gitops_repo.md) and leave it uninitialized.
```

Comment:
We don't need this preq anymore ?

Reply:
I think we don't as  kam is creating the repo using authentication token.